bin/inference/pycbc_inference_plot_inj_intervals

```diff
@@ -72,7 +70,7 @@ fig = plt.figure()
 ax = fig.add_subplot(111)
 
 # calculate the expected percentile for each injection and plot
-for param,label in zip(input_parameters, labels):
+for param,label in zip(parameters, labels):
```

Comment:
`labels` is a dict now, not a list, so this should be:
```
for param in parameters:
    label = labels[param]
```

Reply:
Thanks for pointing this. Made necessary changes and initiated pull request again.